src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/AceEditorBackgroundLinkHighlighter.java

```diff
@@ -762,4 +760,7 @@ public AnchoredRange getRange()
    private FilesServerOperations server_;
    private UserPrefs userPrefs_;
    private static final ViewsSourceConstants constants_ = GWT.create(ViewsSourceConstants.class);
+
+   // constants
+   static Pattern kIssueLinkPattern = Pattern.create("((?:github|gitlab)::)?([-a-zA-Z0-9.]+/[-a-zA-Z0-9.]+)?#([0-9]+)");
```

Comment:
```suggestion
   private static final Pattern ISSUE_LINK_PATTERN = Pattern.create("((?:github|gitlab)::)?([-a-zA-Z0-9.]+/[-a-zA-Z0-9.]+)?#([0-9]+)", "");
```

We usually use shouty-case for static final Java objects.

I'm also guessing that we want the second parameter to be `""` as otherwise the pattern will get the `/gm` flag and be stateful (correct me if I'm wrong):

https://github.com/rstudio/rstudio/blob/aa90ba6df9e8a8647c8ca007ba59447b6401615e/src/gwt/src/org/rstudio/core/client/regex/Pattern.java#L28-L30

Reply:
rolled back to `Pattern.create()` as using `""` as the second parameter made the whole application still for some reason.